content/tutorial/02-sveltekit/02-routing/03-loading-data/app-a/src/routes/api/+server.js

```diff
@@ -0,0 +1,5 @@
+// We'll get to +server.js in a later chapter
+
+export function GET() {
+	return new Response('Hello from your API');
```

Comment:
im thinking maybe add a 5 second delay, so, we can show to the user that it is obvious that the page loads with a `<p>undefined</p>` then 5 seconds later turn into data.

as compared to takes 5 seconds to see the complete rendered page.

what do you think?

Reply:
I think that would make the SvelteKit-version look worse ("oh I have to wait 5 seconds until _anything_ is visible?") - I can see `undefined` flashing in the output, so I'm ok with that.

Reply:
fair enough